scripts/benchmark.js

```diff
@@ -12,14 +12,28 @@ const babel  = require("babel-core");
 const zlib   = require("zlib");
 const fs     = require("fs");
 const path   = require("path");
+const Command = require("commander").Command;
```

Comment:
Sorry, I didn't realize commander was pulled in by another package.  Have added commander to devDependencies in package.json